pkg/api/server/server.go

```diff
@@ -125,6 +129,14 @@ func (server *DefaultServer) QueryRateLimiter() (*rate.Limiter, bool) {
 	return server.queryRateLimiter, server.queryRateLimiter != nil
 }
 
+func (server *DefaultServer) registerInfoRoutes() {
+	// make sure these endpoints don't interfere with the standard API path
+	infoGroup := server.router.Group("/-")
+	infoGroup.GET("/info", api.ServiceInfoHandler(server.serviceName))
```

Comment:
Nitpick: You could use consts for the routes (you're already using the strings up top in the definition of the `...Route` consts...